bindgen/codegen/impl_debug.rs

```diff
@@ -194,8 +194,14 @@ impl<'a> ImplDebug<'a> for Item {
                             self.#name_ident
                                 .iter()
                                 .enumerate()
-                                .map(|(i, v)| format!("{}{:?}", if i > 0 { ", " } else { "" }, v))
-                                .collect::<String>()
+                                .fold(String::new(), |mut output, (i, v)| {
```

Comment:
maybe an imperative approach is clearer here?

Reply:
fixed